v6/v6_interface.ml

```diff
@@ -28,9 +28,9 @@ exception Missing_connection_details
 exception License_checkout_error of string
 
 (* dbg_str -> requested edition -> current params ->
-   new edition * new xapi params * new additional params *)
```

Comment:
Well, let's do that right now then, otherwise it will not happen. I agree that using a record will make the interface clearer.